ocaml/xapi/certificates.ml

```diff
@@ -19,6 +19,16 @@ module D = Debug.Make (struct let name = "certificates" end)
 
 open D
 
+let () = Mirage_crypto_rng_unix.initialize ()
+
+(* Certificate locations:
+  * a) stunnel external             = /etc/xensource/xapi-ssl.pem
+  * b) stunnel SNI (internal)       = /etc/xensource/xapi-pool-tls.pem
+  * c) user trusted cert folder     = /etc/stunnel/certs/
+  * d) internal trusted cert folder = /etc/stunnel/certs-pool/
+  * e) appliance trusted bundle     = /etc/stunnel/xapi-stunnel-ca-bundle.pem
+  * f) host-in-pool trusted bundle  = /etc/stunnel/xapi-pool-ca-bundle.pem *)
```

Comment:
e) and f) are created by a script from c) and d). Are we calling this script (update-ca-bundle.sh) whenever we add or remove certs?

Reply:
e) is called by existing API calls installing CA certificates in `ocaml/xapi/certificates`
f) is not done yet and it must happen on pool join and on first-boot (startup)

Reply:
I'll mention in the comment that these are auto-generated (however I think actually adding that functionality is out of scope for this PR)